packages/@lwc/integration-karma/test/light-dom/scoped-styles/index.spec.js

```diff
@@ -82,4 +84,13 @@ describe('Light DOM scoped CSS', () => {
             getComputedStyle(elm.shadowRoot.querySelector('x-light-child div')).color
         ).not.toEqual('rgb(255, 0, 0)');
     });
+
+    it('properly scopes pseudo-elements', () => {
+        const elm = createElement('x-pseudo-parent', { is: PseudoParent });
+        document.body.appendChild(elm);
+        const ids = extractDataIds(elm);
+
+        expect(getComputedStyle(ids.parentDiv, '::after').color).toEqual('rgb(255, 0, 0)');
+        expect(getComputedStyle(ids.childDiv, '::after').color).toEqual('rgb(0, 0, 0)');
+    });
```

Comment:
Test fails before the fix, succeeds after. Before the fix, the colors are actually reversed. 🤦